src/v/cluster/service.cc

```diff
@@ -450,6 +450,14 @@ void clear_partition_revisions(node_health_report& report) {
         }
     }
 }
+
+void clear_partition_sizes(node_health_report& report) {
+    for (auto& t : report.topics) {
+        for (auto& p : t.partitions) {
+            p.size_bytes = 0;
```

Comment:
perhaps we should use `size_t(-1)` as a sentinel value? 0 value is kind of valid. @mmaslankaprv what do you think?